INSTALL.md

```diff
@@ -1,44 +1,48 @@
 # Install
 
-## Quick start using the remote Astrobee docker images
+## System requirements
 
-*The following has been tested on native Ubuntu systems using X11 (the default). Please see [these ROS pages](http://wiki.ros.org/docker/Tutorials#Tooling_with_Docker) for more resources.*
+Ubuntu 20.04 is the preferred host OS for most Astrobee developers to use.
 
-Make sure you have Docker installed in your Ubuntu system following the [installation instructions](https://docs.docker.com/engine/install/ubuntu/) and [post-installation steps for Linux](https://docs.docker.com/engine/install/linux-postinstall/).
+Here are the available host OS options with development roadmap details:
+- Ubuntu 20.04: This is the preferred host OS for most Astrobee developers to use. The Astrobee Facility team is currently preparing to upgrade the robots on ISS from Ubuntu 16.04 to Ubuntu 20.04, but we aren't yet ready to announce a deployment date for that upgrade.
+- Ubuntu 18.04: We are not aware of any current robot users that still need Ubuntu 18.04 support, and expect to discontinue support in the near future. New users should not select this host OS.
+- Ubuntu 16.04: The Astrobee robot hardware on ISS currently runs Ubuntu 16.04. Only developers with NASA internal access can cross-compile software to run on the robot, and must use 16.04 for that. Most developers shouldn't need to work with 16.04, especially when just getting started. Support will eventually be discontinued after the robot hardware on ISS is upgraded to Ubuntu 20.04.
 
-For some systems (with discrete graphics cards), you may need to install [additional software](http://wiki.ros.org/docker/Tutorials/Hardware%20Acceleration).
+Graphical interfaces will perform best if your host OS is running natively (not in a virtual machine).
 
-``` bash
-git clone https://github.com/nasa/astrobee.git
-cd astrobee
-./scripts/docker/run.sh --remote
-```
+Your host OS must have an X11 server installed if you want to use graphical applications, even if you are developing inside a Docker container (the X11 application running inside the container will forward its interface to the host's X11 server). X11 comes with Ubuntu Desktop by default.
 
-For more options, see: \ref install-docker.
+If you plan to develop inside Docker, see [this page on using ROS with Docker](http://wiki.ros.org/docker/Tutorials#Tooling_with_Docker) for more details.
 
-## Building the code natively
+## Option 1: Install inside a Docker container
```

Comment:
Technically not installing anything xD I suggest replacing with "Using", but optional

Reply:
Oh, no, I disagree. This does download a pre-built Docker image to the local machine, which is a form of installation. Plus I felt like I should write it that way on the INSTALL page, after all :)